samples/msal-node-extensions/index.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const express = require("express");
+const msal = require('@azure/msal-node');
+const extensions = require("msal-node-extensions");
+const { promises: fs } = require("fs");
+const path = require("path");
+
+const SERVER_PORT = process.env.PORT || 3000;
+const cachePath = path.join(__dirname, "./cache.json");
+
+extensions.FilePersistenceWithDataProtection.create(cachePath, extensions.DataProtectionScope.LocalMachine)
+    .then((filePersistence) => {
+
+            const publicClientConfig = {
+                auth: {
+                    clientId: "99cab759-2aab-420b-91d8-5e3d8d4f063b",
+                    authority: "https://login.microsoftonline.com/90b8faa8-cc95-460e-a618-ee770bee1759",
+                    redirectUri: "http://localhost:3000/redirect",
+                },
+                cache: {
+                    cachePlugin:  new extensions.PersistenceCachePlugin(filePersistence)
+                },
+            };
+            const pca = new msal.PublicClientApplication(publicClientConfig);
+            const msalCacheManager = pca.getCacheManager();
+
+            // Create Express App and Routes
+            const app = express();
+
+            app.get('/', (req, res) => {
+                const authCodeUrlParameters = {
+                    scopes: ["openid"],
+                    redirectUri: "http://localhost:3000/redirect",
+                };
+
+                // get url to sign user in and consent to scopes needed for application
+                pca.getAuthCodeUrl(authCodeUrlParameters).then((response) => {
+                    res.redirect(response);
+                }).catch((error) => console.log(JSON.stringify(error)));
```

Comment:
I would send back the error in the response, in addition to printing it to the console.